packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -37,26 +37,6 @@ export const EVENTS = {
   REMOTE_MEDIA_STOPPED: 'REMOTE_MEDIA_STOPPED',
 };
 
-/**
- * Given a media type, return a simplified version of it.
- *
- * @param {String} mediaType - The complex media type (i.e. `video-send-2`).
- * @returns {string | null} A string of simplified media type or null.
- */
-const getSimplifiedMediaType = (mediaType) => {
-  if (mediaType.includes('share')) {
-    return STATS.SHARE_CORRELATE;
-  }
-  if (mediaType.includes('audio')) {
-    return STATS.AUDIO_CORRELATE;
-  }
-  if (mediaType.includes('video')) {
-    return STATS.VIDEO_CORRELATE;
```

Comment:
so now I think we shouldn't be using these `STATS.*_CORRELATE` constants anywhere anymore, right? When I search the code I see that `STATS.AUDIO_CORRELATE` is still used in a few places, can you check them?

Reply:
Right.  I removed the `STATS.*_CORRELATE` constants and adjusted the references to `STATS.AUDIO_CORRELATE`.